app/locales/en.json

```diff
@@ -135,5 +135,14 @@
   "Close contact with someone who has tested positive": "Close contact with someone who has tested positive",
   "Tested negative": "Testet negative",
   "Map": "Map",
-  "API": "API"
+  "API": "API",
+  "I agree to set a cookie so I can change my health condition later e.g. to report recovery/getting sick": "I agree to set a cookie so I can change my health condition later e.g. to report recovery/getting sick",
+  "required": "required",
+  "recommended": "recommended",
+  "Below is your personal health information. If you want to register another person on this computer, save or write down your personal link": "Below is your personal health information. If you want to register another person on this computer, save or write down your personal link",
```

Comment:
Did someone translate these texts to Norwegian/Dutch as well?

Reply:
Not yet! Is there a way to remove the `\n        ` stuff from the keys?

Reply:
Hmm, I don't know. It is pretty fragile this way, since indentation changes probably will make the keys invalid.

Reply:
Fixed now.